### PR TITLE
#170947368 Setup an empty react project

### PR DESCRIPTION
**Pull request by blaise82**
#### What does this PR do?
create a react project from scratch using Webpack 4, Babel, and the Airbnb linting option.

#### Description of Task to be completed?

- [x] add .gitignore file
- [x] Add .babelrc file
- [x] Add islintignore file
- [x] create web pack.config file
- [x] create src folder
- [x] setup webpack
- [x] setup babel
- [x] eslint

#### How should this be manually tested?

-  git clone the repo and checkout branch ch-setup-react-project-170947368
- run npm install to install dependencies.
- run npm run dev to run the app /dev mood

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/170947368

#### Screenshots (if appropriate)
Screen Shot 2020-04-15 at 01 47 34

[Finishes #170947368]

### CODE REVIEW

**Review by Moise1 — CHANGES_REQUESTED** [review]
Hey @blaise82 , please fix the **no-new-line** eslint errors at the end of the lines in all files.

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+				use: [ 'babel-loader' ]
+			}
+		]
+	},
+	plugins: [
+		new HtmlWebpackPlugin({
+			template: './src/index.html'
+		}),
+		new webpack.DefinePlugin({
+			PRODUCTION_API: JSON.stringify('https://octopus-bn-backend-staging.herokuapp.com//api/v1/')
```
Comment (Moise1) [inline webpack.config.js]: Again, @blaise82 , I think you mistakenly left a second slash after the **.com** in the Heroku link. Please double check.

Comment (blaise82) [reply webpack.config.js]: ohh let m e fix that

**Comment by blaise82** [comment]
> Hey @blaise82 , please fix the **no-new-line** eslint errors at the end of the lines in all files.

what

**Comment by blaise82** [comment]
> Hey @blaise82 , please fix the **no-new-line** eslint errors at the end of the lines in all files.

what's issue exactly.
u don't want those errors or they are not showing?

**Review by Moise1 — APPROVED** [review]

**Review by hakoemmy — APPROVED** [review]
@blaise82 Good job.

**Review by AbdoulNuru — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "octopus-bn-frontend",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack-dev-server --mode development --open --hot --port 3030",
```
Comment (AJAkimana) [inline package.json]: Use environment variables

File: package.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    "build": "webpack --mode production",
+    "start": "serve -s ./build",
```
Comment (AJAkimana) [inline package.json:9]: Start server after running **build**

File: package.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+  },
+  "dependencies": {
+    "@material-ui/core": "^4.9.10",
+    "@material-ui/icons": "^4.9.1",
+    "eslint-config-react-app": "^5.2.1",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
+    "serve": "^11.3.0"
+  }
+}
```
Comment (AJAkimana) [inline package.json]: Add new line

**Review by AJAkimana — COMMENTED** [review]
> Work on the inline comments and put the HTML index file into static files

Sorry, may you shade more light on the comments.

If I got it right about the second statement you meant like having folder public and put the .html file in?

**Comment by AJAkimana** [comment]
> > Work on the inline comments and put the HTML index file into static files
> 
> Sorry, may you shade more light on the comments.
> 
> If I got it right about the second statement you meant like having folder public and put the .html fine in?

You are right, that's what I meant

**Pull request merged**